vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java

```diff
@@ -138,6 +140,26 @@
   public static final String LOCATION = "location"; //NON-NLS
   public static final String USE_GRID_LOCATION = "useGridLocation"; //NON-NLS
 
+  public static boolean showOthers = false; // Whether we also show other decks/stacks when changing the position of one
```

Comment:
This is just for the editor, right? Why is this a member of `SetupStack` rather than being a non-static member of some the editor class?

Reply:
It is just for the editor, correct. This situation *only* happens when configuring a SetupStack, and so it didn't seem appropriate to put it somewhere broader?

Reply:
Oh and it's static because I'm in one StackConfigurer (inner class of SetupStack) and I need to tell *all the others that exist* that they need to behave themselves. 

I guess you're saying I could tell e.g. GameState that we're in the middle of a configurer draw and so therefore behave differently. And then read that from a non-static member/method of GameState. It seemed cleaner to me to keep all the code in the same file, but I can do that if you'd prefer.